examples/meta/generator/targets/cpp.json

```diff
@@ -0,0 +1,61 @@
+{
+    "Program": "#include <shogun/base/init.h>\n#include <shogun/io/SGIO.h>\n#include <shogun/features/DenseFeatures.h>\n#include <shogun/labels/MulticlassLabels.h>\n#include <shogun/multiclass/KNN.h>\n#include <shogun/io/CSVFile.h>\n#include <shogun/distance/EuclideanDistance.h>\n\n${dependencies}\n\nusing namespace shogun;\n\nint main(int, char*[]) {\n    init_shogun_with_defaults();\n\n$program\n    exit_shogun();\n    return 0;\n}\n",
```

Comment:
@sorig you mentioned the explicit imports in python. We need the same thing for c++.
We have a problem here though: the directory structure of shogun. Basically, we somehow need to infer the include path from the type. Any ideas for doing this in an elegant way?